specs/core/1_custody-game.md

```diff
@@ -155,7 +155,7 @@ This document details the beacon chain additions and changes in Phase 1 of Ether
     'inclusion_epoch': Epoch,
     'data_root': Hash,
     'chunk_count': 'uint64',
-    'chunk_bits_merkle_root': Hash,
+    'chunk_bits_merkle_root': 'bytes32',
```

Comment:
In my head we got rid of the `Hash` type, am I correct in this?

Reply:
I think so. Although there is a suggestion to go back to using explicit aliases in #667, which I support, for now it's just "bytes32".